cmd/vulcan-trivy/vulcan-trivy.go

```diff
@@ -33,6 +33,11 @@ import (
 const (
 	vulnCVETruncateLimit = 10
 	DefaultDepth         = 1
+	// trivyTimeout defines the value that will be passed to the trivy CLI via
+	// the `--timeout` flag. The value should be bigger than the check timeout
+	// defined in the manifest, to ensure the check will have a `TIMEOUT`
+	// status when the execution takes longer than expected.
+	trivyTimeout = `2h`
```

Comment:
I would use double quotes here. I usually reserve the use of raw string literals to strings that contain double quotes, back-slashes, regular expressions, etc.

```suggestion
	trivyTimeout = "2h"
```